src/common/form/fields/_textField.scss

```diff
@@ -0,0 +1,9 @@
+.app-TextField {
+  &__label {
+
```

Comment:
Why you have some empty one like this?

Reply:
Removed the empty label and input selectors